<launch>

    <arg name="isTest" default="true"/>

    <arg name="hdl_topic" default="hdl_tf"/>
    <arg name="odom_topic" default="odometry/global"/>
    <arg name="matching_err_tol" default="0.05"/>
    <arg name="inlier_fraction_tol" default="0.95"/>

    <include file="$(find map_matching_localization)/launch/test.launch" if="$(arg isTest)"/>

    <node pkg="map_matching_localization" type="hdl_tf.py" name="hdl_tf_node" output="screen">
        <param name="hdl_topic" value="$(arg hdl_topic)"></param>
        <param name="odom_topic" value="$(arg odom_topic)"></param>
        <param name="matching_err_tol" value="$(arg matching_err_tol)"></param>
        <param name="inlier_fraction_tol" value="$(arg inlier_fraction_tol)"></param>
    </node>

</launch>
